config/locales/diaspora/en.yml

```diff
@@ -683,6 +683,7 @@ en:
         subject: "%{name} has mentioned you on diaspora*"
         limited_post: "You were mentioned in a limited post."
     private_message:
+        subject: "Diaspora: Private Message"
```

Comment:
The sender of the e-mail is diaspora*, I'm not sure it is necessary to add it there.